test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/BusinessAccountingMethodViewSpec.scala

```diff
@@ -141,18 +165,20 @@ class BusinessAccountingMethodViewSpec extends ViewSpec {
     ).mustHaveGovUkErrorNotificationMessage(emptyError)
   }
```

Comment:
can you add a test for back link for two different scenarios:
1. when showing 
2. not showing

Reply:
The tests are on line 64 and line 69.